joomla/joomla-cms
# [5.0] Rework the com_messages message view

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/42124 .

### Summary of Changes
Rework of the HTML structure towards something geared towards more input'less elements. Physically separates the invisible form from the rendered HTML content.

### Testing Instructions
Whilst this ensures the view looks better in dark mode. It obviously changes the structure in light mode too. This too should be validated. Check with longer and shorter message lengths.

### Actual result BEFORE applying this Pull Request
![dark-private-messages](https://github.com/joomla/joomla-cms/assets/368084/e7c89ef8-338e-4a29-aa9b-2b1fdf37ab03)

### Expected result AFTER applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1986000/25853664-fadb-4340-ba04-cd2a70990fab)

### Link to documentations
Please select:
